### ISSUE TITLE
Support "cleanup" after all tests in a "Describe" run

### ISSUE THREAD

**Issue by theficus** [pester/Pester#298]
I realize there's ways to work around this, for example by just putting some code at the end of a .Tests.ps1 fixture but it would be nice if this were formalized into the Describe { } flow.

MSTest has a concept of a static cleanup method that runs after all tests within a group run. It would be nice if there was a similar formal concept within a Describe or Context that would run a set of cleanup instructions only after every It or Context block ran. Basically like a one-shot "AfterEach".

**Comment by dlwyatt** [comment]
That's already done.  You want the `AfterAll` command.  :)  You can see this in the Describe function itself: there's a try/catch/finally block, and the first thing that happens in finally is Invoke-TestGroupTeardownBlocks (which are the internal name for the contents of any `AfterAll` blocks in the script.)

**Comment by theficus** [comment]
Derp. I must have missed this in the documentation. Thanks!

**Comment by dlwyatt** [comment]
Doesn't look like I added that to the wiki yet, though it should be in the help in the module itself.  I'm kind of terrible about that; will get the wiki updated soon.
